baseband/mark5b/base.py

```diff
@@ -172,40 +172,40 @@ def write_frame(self, data, header=None, bps=2, valid=True, **kwargs):
 class Mark5BStreamReader(VLBIStreamReaderBase, Mark5BFileReader):
     """VLBI Mark 5B format reader.
 
-    This wrapper is allows one to access a Mark 5B file as a continues series
-    of samples.  Note that possible gaps in the data stream are not filled in.
+    Allows access a Mark 5B file as a continues series of samples.  Note that
```

Comment:
I'm actually not sure why we mention the gaps here and nowhere else. Would suggest to remove it.